pkgs/development/libraries/modern-cpp-kafka/default.nix

```diff
@@ -0,0 +1,48 @@
+{ rdkafka, stdenv, lib, fetchFromGitHub, cmake, zlib, breakpointHook, boost, gtest, rapidjson, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "modern-cpp-kafka";
+  version = "v2023.03.07";
+
+  src = fetchFromGitHub {
+    repo = "modern-cpp-kafka";
+    # owner = "morganstanley";
+    # rev = version;
+    # hash = "sha256-7hkwM1YbveQpDRqwMZ3MXM88LTwlAT7uB8NL0t409To=";
+    #
+    # Patch build to avoid overwriting include path
+    # PR: https://github.com/morganstanley/modern-cpp-kafka/pull/221
+    owner = "ditsuke";
+    rev = "build/avoid-overwriting-include-path";
+    hash = "sha256-fRKIDaDkjBHKKrBUj/TfjQpvF1jMQacj5AXB28cLWxE=";
+  };
+
+  cmakeFlags = [
+    "-DLIBRDKAFKA_INCLUDE_DIR=${rdkafka.out}/include"
+    "-DGTEST_LIBRARY_DIR=${gtest.out}/lib"
+    "-DGTEST_INCLUDE_DIR=${gtest.dev}/include"
+    "-DRAPIDJSON_INCLUDE_DIRS=${rapidjson.out}/include"
+    "-DCMAKE_CXX_FLAGS=-Wno-uninitialized"
+  ];
+
+  buildInputs = [ boost ];
+  propagatedBuildInputs = [ rdkafka ];
+  testInputs = [ gtest rapidjson ];
+  nativeBuildInputs = [ cmake ];
+
+  buildPhase = ''
+    cmake .
+    cp -r . $out
+  '';
+
+  outputs = [ "out" ];
+
+
+  meta = with lib; {
+    homepage = "https://github.com/morganstanley/modern-cpp-kafka";
+    description = "A C++ API for Kafka clients (i.e. KafkaProducer, KafkaConsumer, AdminClient)";
+    license = licenses.asl20;
+    platforms = platforms.unix;
```

Comment:
Can you add yourself as a maintainer for this package ( via a separate commit ) ?

Reply:
done